src/Stripe.net/Services/Customers/CustomerUpdateOptions.cs

```diff
@@ -24,16 +24,16 @@ public class CustomerUpdateOptions : BaseOptions, IHasMetadata
         public long? Balance { get; set; }
 
         [JsonProperty("coupon")]
-        public string Coupon { get; set; }
+        public AnyOf<string, EmptyParam> Coupon { get; set; }
 
         [JsonProperty("default_source")]
-        public string DefaultSource { get; set; }
+        public AnyOf<string, EmptyParam> DefaultSource { get; set; }
```

Comment:
I am fairly sure you can never unset a default source. If the openapi spec says otherwise, the spec is wrong.

Reply:
I copied this from stripe-java, so the spec is likely wrong.